app.rb

```diff
@@ -74,32 +74,30 @@ def handle_event(event:, context:)
 def get_holding(event)
```

Comment:
Generally I prefer to move methods and functions that don't have to do with the control flow or lifecycle of the lambda to their own modules. I think it makes it a little cleaner and easier to maintain.

That is definitely my own preference though, I just wanted to raise this as a topic of conversation, especially as we're starting to move towards deploying this stuff into production.